server/service/osquery_utils/queries.go

```diff
@@ -1040,6 +1040,18 @@ SELECT
   '' AS installed_path
 FROM rpm_packages
 UNION
+SELECT
```

Comment:
Ah, just realized this will break software ingestion for every customer using vanilla osquery or older versions of fleetd.

We'll need to use the VS Code extensions approach. Where we run the query as an additional query (with the discovery query) and append the extra results to the results of the main query.